gitlab-pipeline/stage/test.yml

```diff
@@ -201,12 +206,17 @@ test:backend-integration:azblob:enterprise:
     - apk --update add curl jq sysstat hdparm
 
     # Start dockerd in the background
-    - /usr/local/bin/dockerd &
+    # The entrypoint script additionally uses docker-init as workarround of known
+    # containerd issue - https://github.com/docker-library/docker/issues/318
+    - export TINI_SUBREAPER=true
```

Comment:
Can we add a pointer to the documentation for this variable? I found this:
* https://github.com/krallin/tini#subreaping

Reply:
Yes, this is correct doc 👍